lib/handlers/TwilioHandler.js

```
@@ -189,6 +193,64 @@ exports.handleConference = function (req, res, next) {
+                foundApplication = application;
+                return q.nfcall(applicationController.findPrimary, foundApplication._id);
+            })
+            .then(function (primary) {
+                if (primary && screening != 'primary') {
+                    twilioClient.messages.create({
+                        from: req.param('From'),
+                        to: primary.Primary,
+                        body: message
+                    }, function(err, message) {
+                        console.log('SMS Error: ' + err);
+                        if (message)
+                            console.log(message.sid);
+                    });
+                }
+
+                return q.nfcall(applicationController.findSecondary, foundApplication._id);
+            })
+            .then(function (secondary) {
+                if (secondary && screening != 'secondary') {
+                    twilioClient.messages.create({
+                        from: req.param('From'),
+                        to: secondary.Primary,
+                        body: message
+                    }, function(err, message) {
+                        console.log('SMS Error: ' + err);
+                        if (message)
+                            console.log(message.sid);
+                    });
+                }
+
+                return q.nfcall(staffController.findByID, foundApplication.Fallback._id);
+            })
+            .then(function (fallback) {
+                if (fallback && screening != 'fallback') {
+                    twilioClient.messages.create({
+                        from: req.param('From'),
+                        to: fallback.Primary,
+                        body: message
+                    }, function(err, message) {
+                        console.log('SMS Error: ' + err);
+                        if (message)
+                            console.log(message.sid);
+                    });
+                }
+            });
```

I did not see any error handling in this promise chain. This means that if a database lookup fails or some other error occurs, it will be silently ignored. At the very least, `.done()` should be used, but ideally we would do something a bit more elegant.